src/ziggurat/init.clj

```diff
@@ -143,12 +143,17 @@
                            (shutdown-agents))
             "Shutdown-handler")))
 
+(declare StreamRoute)
```

Comment:
can we leave it? since it does not affect the behavior of the code. I think its a good practice to declare some vars/symbols first before using it....